packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -59,6 +59,7 @@ export interface RawRumViewEvent {
     firstContentfulPaint?: number
     firstInputDelay?: number
     cumulativeLayoutShift?: number
+    customTimings: ViewCustomTimings
```

Comment:
This field should be added in rum-events-format as well to update rumEvent.types